docs: Sameness groups GA (#19103)
hashicorp/consul: website/content/docs/k8s/connect/cluster-peering/usage/create-sameness-groups.mdx

Reviewer: I think my comments from connect `website/content/docs/connect/cluster-peering/usage/create-sameness-groups.mdx` apply here as well

Before creating a sameness group, take the following actions to prepare your network:

#### Check Consul namespaces and service naming conventions

> Sameness groups are defined at the partition level. Consul assumes these partitions have identical configurations, including identical service names and identical Consul namespaces. This behavior occurs even when two partitions in the group contain functionally different services that share a common name and namespace. For example, if distinct services both named `api` were registered to two members of a sameness group, it could lead to errors because requests may be sent to the incorrect service.

Reviewer 2: ```suggestion
Sameness groups are defined at the partition level. Consul assumes all partitions in the group have identical configurations, including identical service names and identical Consul namespaces. This behavior occurs even when two partitions in the group contain functionally different services that share a common name and namespace. For example, if distinct services both named `api` were registered to different members of a sameness group, it could lead to errors because requests may be sent to the incorrect service.
```
For consistency with the VM version

apiVersion: consul.hashicorp.com/v1alpha1
Kind: ExportedServices
metadata:
  name: product-group-export
spec:
  services:
    - name: api
      namespace: store
      consumers:
        - samenessGroup: product-group
```

</CodeBlockConfig>

</CodeTabs>

For more information about exporting services, including examples of CRDs that export multiple services at the same time, refer to the [exported services configuration entry reference](/consul/docs/connect/config-entries/exported-services).

After you create each exported services configuration entry, apply it to the Consul server with the following CLI command:

```shell-session
$ kubectl apply -f product-group-export.yaml
```

#### Export services for cluster peers and sameness groups as part of the same workflow

> If you separately follow the steps in this documentation to create a cluster peering connection between two partitions and then add those partitions to a sameness group, you write and apply two separate exported services CRDs: one to export services to the peer, and a second to export services to other members of the group.

Reviewer 2: ```suggestion
Creating a cluster peering connection between two partitions and then adding the partitions to a sameness group requires that you write and apply two separate exported services CRD. One CRD exports services to the peer, and a second exports services to other members of the group.
```
Same as the VM suggestion

> If you are peering clusters with the goal of creating a sameness group, it is possible to combine these workflows by using a single exported services CRD.

Reviewer 2: Just flagging this section for you to update if you want to implement the feedback on the VM version.
